providers/fips/fipsprov.c

```diff
@@ -384,6 +384,7 @@ static const OSSL_ALGORITHM fips_kdfs[] = {
       ossl_kdf_tls1_3_kdf_functions },
     { PROV_NAMES_SSKDF, FIPS_DEFAULT_PROPERTIES, ossl_kdf_sskdf_functions },
     { PROV_NAMES_PBKDF2, FIPS_DEFAULT_PROPERTIES, ossl_kdf_pbkdf2_functions },
+    { PROV_NAMES_SNMPKDF, "provider=default", ossl_kdf_snmpkdf_functions },
```

Comment:
`FIPS_DEFAULT_PROPERTIES` rather than `"provider=default"`.
This isn't the default provider after all.

I'm not sure what policy we should adopt for items that can be FIPS approved but aren't in any of our current validations.

Reply:
Changed to FIPS_DEFAULT_PROPERTIES